src/core_plugins/kibana/public/management/sections/objects/_objects.js

```diff
@@ -164,8 +164,18 @@ uiModules.get('apps/management')
           notify.error('The file could not be processed.');
         }
 
-        return Promise.map(docs, function (doc) {
-          const service = find($scope.services, { type: doc._type }).service;
+        return Promise.map(docs, (doc) => {
+          const { service } = find($scope.services, { type: doc._type }) || {};
+
+          if (!service) {
+            const msg = `Skipped import of "${doc._source.title}" (${doc._id}), ` +
```

Comment:
Are you concatenating two template literals here because of line length rules? Seems weird to me.